### PR TITLE
Fix build by adding missing dep

### PR DESCRIPTION
**Pull request by jeongseok-meta**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

Fix build failure by adding missing dep, `setuptools`

https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=1076759&view=logs&j=656edd35-690f-5c53-9ba3-09c10d0bea97&t=986b1512-c876-5f92-0d81-ba851554a0a3&l=788

```
"/home/conda/feedstock_root/build_artifacts/easydict_1731195743731/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pl/lib/python3.13/importlib/__init__.py", line 88, in import_module
2024-11-09T23:44:21.0802273Z     return _bootstrap._gcd_import(name[level:], package, level)
2024-11-09T23:44:21.0803050Z            ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-11-09T23:44:21.0803469Z   File "<frozen importlib._bootstrap>", line 1387, in _gcd_import
2024-11-09T23:44:21.0803909Z   File "<frozen importlib._bootstrap>", line 1360, in _find_and_load
2024-11-09T23:44:21.0804352Z   File "<frozen importlib._bootstrap>", line 1310, in _find_and_load_unlocked
2024-11-09T23:44:21.0804810Z   File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
2024-11-09T23:44:21.0805251Z   File "<frozen importlib._bootstrap>", line 1387, in _gcd_import
2024-11-09T23:44:21.0805695Z   File "<frozen importlib._bootstrap>", line 1360, in _find_and_load
2024-11-09T23:44:21.0806142Z   File "<frozen importlib._bootstrap>", line 1324, in _find_and_load_unlocked
2024-11-09T23:44:21.0806659Z ModuleNotFoundError: No module named 'setuptools'
```

### CODE REVIEW

**Comment by jeongseok-meta** [comment]
The last lint suggestions seems false alarm, as they are already resolved by [eea2a88](https://github.com/conda-forge/easydict-feedstock/pull/11/commits/eea2a8875e50e5fc4d9d5e92952e4033ff8654b7).

**Comment by jeongseok-meta** [comment]
@hmaarrfk ready for review!

File: recipe/meta.yaml [hunk 1]
```diff
@@ -45,6 +54,7 @@ about:
   license: LGPL-3.0-only
   license_file: LICENSE
   summary: Access dict values as attributes (works recursively).
+
 extra:
   recipe-maintainers:
     - zhaofeng-shu33
```
Comment (hmaarrfk) [inline recipe/meta.yaml:60]:
```suggestion
    - zhaofeng-shu33
    - jeongseok-meta
```

**Comment by hmaarrfk** [comment]
@conda-forge-admin please rerender

**Pull request merged**